### ISSUE TITLE
What is meas_type?

### ISSUE THREAD

**Issue by johannjacobsohn** [jgliss/pyplis#20]
I cant figure out what meas_type refers to. Measurement Type? But then what's a measurement type?

`scripts/ex0_2_camera_setup.py` indicates meas type is unimportant, yet I am pretty sure files cannot be separated by filter_id if meas_type is not set.

> Param meas_type_acro is only of importance if a meas type (e.g. M -> meas, C -> calib ...) is explicitely specified in the file names (not the case for ECII camera but for the HD camera, see specifications in file cam_info.txt for more info)

Also, I don't really understand what's meant by "acronym" in `pyplis.utils.Filter` and other places. Maybe the wording could be improved and clarified.

**Comment by johannjacobsohn** [comment]
That is why you need the wrapping `(?P<meas_type>...)` in
`r'^.*_(?P<date>.*)_(?P<meas_type>(?P<filter_id>.*))_.*'`
https://github.com/jgliss/pyplis/pull/18/files#diff-b7749735e931188e9badac2c809f8837R37

**Comment by jgliss** [comment]
I agree that the way this is being solved is a little cumbersome and I appreciate your efforts to make it more flexible. The `meas_type` attribute was introduced for the naming convention from the group in Heidelberg which has encoded, in addition to the filter type, a measurement type, that is, a character that specifies whether the camera is looking into the plume (M), or taking sky background images (B) or is performing a cell calibration (C). 

I never used `meas_type` in my data, but I would like to keep it for now, as I would need to do some testing first (need to have a look at your PR #18) and see if the HD cam stuff is still working with your new, flexible solution using regexp's.

Yes, agreed that `acronym` is not the best naming here, maybe `abbr` would be better. I would be happy for backwards-compatible suggestions to rename this parameter.

@johannjacobsohn note that I would like to merge py3 into master soon. 

I leave this open for now.
